damus/ContentView.swift

```diff
@@ -472,20 +477,18 @@ struct ContentView: View {
                 confirm_block = false
                 user_blocked_confirm = true
             }
-            
-            Button("Cancel") {
-                confirm_overwrite_mutelist = false
-                confirm_block = false
-            }
         }, message: {
-            Text("No block list found, create a new one? This will overwrite any previous block lists.")
+            Text("No block list found, create a new one? This will overwrite any previous block lists.", comment: "Alert message prompt that asks if the user wants to create a new block list, overwriting previous block lists.")
         })
-        .alert("Block User", isPresented: $confirm_block, actions: {
-            Button("Block") {
+        .alert(NSLocalizedString("Block User", comment: "Title of alert for blocking a user."), isPresented: $confirm_block, actions: {
+            Button(NSLocalizedString("Cancel", comment: "Alert button to cancel out of alert for blocking a user."), role: .cancel) {
+                confirm_block = false
+            }
+            Button(NSLocalizedString("Block", comment: "Alert button to block a user."), role: .destructive) {
```

Comment:
Swapped order of buttons and marked destructive button.
![Simulator Screen Shot - iPhone 14 Pro - 2023-01-26 at 23 07 26](https://user-images.githubusercontent.com/963907/215007549-10792226-1e94-43c7-8205-e398e3c6c645.png)